### DISCUSSION TITLE
Welcome to auto-clicker-auto-fill Discussions!

### DISCUSSION THREAD

**Discussion by dharmesh-hemaram** [Dhruv-Techapps/acf-docs#372] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xDev5** [comment]
Can this be used for transferring a blog from shopify to any other platform? Fully like automated, or at least mostly automated? Any help would be really appreciated, cuz a lot of money and to do

**Reply by dharmesh-hemaram** [reply]
no sorry our extension do not support for such feature . if it require it can be developed separate extension for it.

**Comment by RajGen23** [comment]
hello i have clicked the button but after that a custom alert comes , how to click on that

**Comment by dharmesh-hemaram** [comment]
inspect ok button and add new action for it .

If its default alert of chrome you can bypass same with alert settings
under configuration settings.

On Tue, Sep 24, 2024 at 6:53 PM RajGen23 ***@***.***> wrote:

> hello i have clicked the button but after that a custom alert comes , how
> to click on that
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Dhruv-Techapps/acf-docs/discussions/372#discussioncomment-10739194>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARPEGTXMRMHQPPCC3HOPXDZYFRWXAVCNFSM6AAAAABHDB6QCWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZTHEYTSNA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> .com>
>


-- 
Regards,
Dharmesh Patel
